src/NServiceBus.RabbitMQ/Config/RabbitMqTransportFeature.cs

```diff
@@ -36,6 +36,8 @@ protected override string GetLocalAddress(ReadOnlySettings settings)
 
         protected override void Configure(FeatureConfigurationContext context, string connectionString)
         {
+            Address.IgnoreMachineName();
```

Comment:
As you suggested, I think we should have an explicit issue describing the bug that this fixes.